x-pack/test/functional/config.js

```diff
@@ -89,6 +89,10 @@ export default async function({ readConfigFile }) {
         '--telemetry.banner=false',
         '--timelion.ui.enabled=true',
         '--xpack.endpoint.enabled=true',
+        '--xpack.actions.enabled=true',
+        '--xpack.alerting.enabled=true',
+        '--xpack.triggers_actions_ui.enabled=true',
+        '--xpack.triggers_actions_ui.createAlertUiEnabled=true',
```

Comment:
Once https://github.com/elastic/kibana/pull/60137 is merged, and master is then merged into this PR, these lines will not be necessary.

Reply:
Thank you!